src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -24,17 +24,19 @@ class AppNavigationController: UINavigationController {
 	private var scrollViewObserver: NSKeyValueObservation?
 	private var defaultScrollEdgeAppearance: UINavigationBarAppearance?
 
-	override func viewDidLoad() {
-		super.viewDidLoad()
+	override func loadView() {
```

Comment:
Is this the right method to do this kind of stuff in? What difference does it make in the behaviour in practice comparing to ```viewDidLoad```?

Reply:
`loadView` vs. `viewDidLoad` is mostly a matter of personal preference. But a general rule of thumb is to use `loadView` when creating view controllers programmatically where as `viewDidLoad` can be used to customize a view loaded from a storyboard/nib/xib. As this controller does not have an underlying storyboard I switched to `loadView`.

https://stackoverflow.com/questions/3423785/what-is-the-difference-between-loadview-and-viewdidload

Reply:
But you're not loading any view, right? :) You just setup an already loaded view.

The [official doc](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview) states that: 
"_You can override this method in order to create your views manually. If you choose to do so, assign the root view of your view hierarchy to the view property. ... Your custom implementation of this method should not call super.
... If you want to perform any additional initialization of your views, do so in the viewDidLoad() method._"

So from this it looks like ```ViewDidLoad``` is appropriate here. And furthermore, it says that ```super``` should not be called if this is used.

I'm asking because I'm trying to figure out what is the fix here, how this switch from ```viewDidLoad``` to ```loadView``` helps us fixing the bug. If it helps, this is my assumption, that it's part of the fix.